providers/gitlab/resources/gitlab.lr

```diff
@@ -121,11 +123,21 @@ gitlab.project.approvalSettings @defaults("approvalsBeforeMerge requirePasswordT
 
 
 // GitLab protected branch
-gitlab.project.repository.protectedBranch @defaults("name allowForcePush") {
+gitlab.project.protectedBranch @defaults("name allowForcePush") {
```

Comment:
Changing this in two parallel PRs is normally not a good idea. We should avoid that one. I recommend we merge https://github.com/mondoohq/cnquery/pull/4372 first and then rebase this PR

Reply:
Ok sure, then let's first merge #4372 and I will rebase and fix the issues. Thanks